index.bs

```diff
@@ -2007,6 +2150,40 @@ object:
 
 </div>
 
+<div algorithm="remote end subscribe steps for script.realmCreated">
+
+The [=remote end subscribe steps=] with [=subscribe priority=] 2, given
+|contexts| and |include global| are:
+
+1. Let |environment settings| be a list of all the [=environment settings objects=]
+   that have their [=execution ready flag=] set.
+
+1. For each |settings| of |environment settings|:
+
+  1. Let |related contexts| be an empty set.
```

Comment:
This looks like a good place to re-use the "get related contexts" algorithm defined in #73.

Reply:
Agreed, but we need to merge that one first (or factor out that part, but it seems like it will be more owrk overall). In any case the logging module will need an update when this merges to buffer log events.

Reply:
Sounds good. Thanks.